gateway-service/src/test/java/org/zowe/apiml/gateway/filters/pre/LocationFilterTest.java

```diff
@@ -20,6 +20,8 @@
 
```

Comment:
As part of our effort to move tests to given_when_then format, I think this is a good opportunity to refactor the test names in this class.